corehq/apps/userreports/views.py

```diff
@@ -104,15 +111,14 @@
 from corehq.apps.userreports.indicators.factory import IndicatorFactory
 from corehq.apps.userreports.models import (
     DataSourceConfiguration,
+    RegistryReportConfiguration,
     ReportConfiguration,
     StaticReportConfiguration,
+    UCRExpression,
     get_datasource_config,
     get_report_config,
     id_is_static,
     report_config_id_is_static,
-    RegistryReportConfiguration,
-    RegistryDataSourceConfiguration,
-    is_data_registry_report,
```

Comment:
Oops, these 2 were removed entirely: https://github.com/dimagi/commcare-hq/pull/31588

@proteusvacuum probably good to figure out how that happened in terms of your local dev process. Do you use a tool for organizing imports?

Reply:
Wow, that's embarrassing. I broke this in [this](https://github.com/dimagi/commcare-hq/commit/758d82213ad47edbbac05e5219871949bb26a403) commit. I have no idea what I did here - when I wrote this I was in the process of switching from emacs to VSCode, so it's entirely possible I just messed up the workflow. 

I just checked out out the preceeding commit, and ran through the actions I think I would have taken (automatically added my new imports, then sorted them with the in-built sorting tool), and it doesn't remove this line, so whatever I did back then to cause this was wrong.